SimulationOutputReport/measure.rb

```diff
@@ -451,6 +452,18 @@ def get_outputs(timeseries_frequency,
       end
     end
 
+    # Dehumidifier
+    outputs[:hpxml_dehumidifier_ids].each do |sys_id|
+      end_use = @end_uses[[FT::Elec, EUT::Dehumidifier]]
+      vars = get_all_var_keys(end_use.variable)
+      ep_output_names = @hvac_map[sys_id]
+      keys = ep_output_names.map(&:upcase)
+      end_use.annual_output_by_system[sys_id] = get_report_variable_data_annual_mbtu(keys, vars)
+      if include_timeseries_end_use_consumptions
+        end_use.timeseries_output_by_system[sys_id] = get_report_variable_data_timeseries(keys, vars, UnitConversions.convert(1.0, 'J', end_use.timeseries_units), 0, timeseries_frequency)
```

Comment:
Since there can only be a single dehumidifier, you don't need to use `annual_output_by_system` and `timeseries_output_by_system`. Just use `annual_output` and `timeseries_output`.